include/private/fnft__nsev_slow.h

```diff
@@ -0,0 +1,73 @@
+/*
+* This file is part of FNFT.  
+*                                                                  
+* FNFT is free software; you can redistribute it and/or
+* modify it under the terms of the version 2 of the GNU General
+* Public License as published by the Free Software Foundation.
+*
+* FNFT is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*                                                                      
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://www.gnu.org/licenses/>.
+*
+* Contributors:
+* Shrinivas Chimmalgi (TU Delft) 2020.
+*/
+
+/**
+ * @file fnft__nsev_slow.h
+ * @brief Auxiliary functions used in fnft_nsev and fnft_nsev_slow.
+ * @ingroup nse
+ */
+
+#ifndef FNFT__NSEV_SLOW_H
+#define FNFT__NSEV_SLOW_H
+
+#include "fnft__nse_discretization.h"
+#include "fnft__akns_scatter.h"
+#include "fnft__errwarn.h"
+
+/**
+ * @brief Refines initial guesses of bound states using Newton's method
+ * 
+ * The function performs Newton's iterations to refine initial guesses of bound states
+ * for the nonlinear Schroedinger equation.
+ * @param[in] D Number of samples
+ * @param[in] q Array of length D, contains samples \f$ q_n\f$ for \f$n=0,1,\dots,D-1\f$
+ * in ascending order (i.e., \f$ q_0, q_1, \dots, q_{D-1} \f$). The values
+ * should be specifically precalculated based on the chosen discretization.
```

Comment:
If you are not already familiar with how the preprocessing works, it might be difficult to figure out from this what to do (and when). Maybe replace the sentence "The values ..." with something like "For certain discretizations, the samples have to be preprocessed. See LINK TO SOMEWHERE WHERE THIS IS EXPLAINED for details."?

Reply:
Link added.